bin/sl-run.js

```diff
@@ -12,10 +12,17 @@ var assert = require('assert');
 var config = require('../lib/config'); // May exit, depending on argv
 var log = config.logger;
 
+if (config.enableTracing) {
+  var tracer = require('../lib/tracer');
+  var traceObject = require('../lib/trace-object');
+  tracer(traceObject.tracerOptions);
+}
+
 var agent = require('../lib/agent');
 var agentOptions = {
   quiet: config.isWorker, // Quiet in worker, to avoid repeated log messages
   logger: config.logger,
+  strongTracer: tracer(),
```

Comment:
Yes, tracer() will return null if config.enableTracing is false.  Agent.prototype.configure will set strongTracer to null.  If this.strongTracer is false, Agent.prototype.transactionLink does not replace the callback.